### PR TITLE
Creates doc on enabling jobs and job hooks - admin

### PR DESCRIPTION
**Pull request by jvanderaa**
# Closes: #3400
# What's Changed

Adds administration page on how to enable Jobs and Job hooks

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Comment by gsnider2195** [comment]
This should probably be linked to from
- `nautobot/docs/models/extras/job.md`
- `nautobot/docs/models/extras/jobbutton.md`
- `nautobot/docs/models/extras/jobhook.md`

**Comment by glennmatthews** [comment]
Also probably should be added to the table-of-contents in `mkdocs.yaml` - if we even need a separate file for this?

**Review by HanlinMiao — APPROVED** [review]
LGTM! IDK if adding some screenshots would be more helpful in this case since there are a decent amount of navigation in the UI.

**Comment by jvanderaa** [comment]
I went down the path that this should be on administering Nautobot. It may fit in the legacy section if you would like.

**Comment by jvanderaa** [comment]
I've got the requests from Gary and Glenn above, but priorities for the week. Do you have a guide that I should be looking at @bryanculver as far as linking so others can find it?

**Comment by jvanderaa** [comment]
I should be getting this updated by next week.

**Comment by jvanderaa** [comment]
Alright, got to the changes. Do we think that we should move the Jobs information out of additional-features and into administration? The docs fit in both spots. As the new addition, and some of the other docs are something only administrators would do.

File: nautobot/docs/additional-features/jobs.md [hunk 1]
```diff
@@ -509,6 +509,27 @@ When a new Job record is created for a newly discovered Job class, it defaults t
...
 
+ By default when a Job is installed into Nautobot it is installed in a disabled state. In order to enable a Job:
+
+ * Navigate to Jobs > Jobs menu
+ * Select a job that has been installed
+ * Select **Edit** button
+ * In the second section titled _Job_, select the **Enabled** checkbox
+ * Select **Update** button at the bottom
+
+### Enabling Job Hooks
```
Comment (glennmatthews) [inline nautobot/docs/additional-features/jobs.md:520]: Do we need to broaden this heading to include Job Buttons?

File: nautobot/docs/models/extras/job.md [hunk 2]
```diff
@@ -24,3 +24,5 @@ Records of this type store the following data as read-only (not modifiable via t
     As presently implemented, after a job is uninstalled, when the database is next refreshed, the corresponding Job database record will *not* be deleted - only its `installed` flag will be set to False. This allows existing `JobResult` and `ScheduledJob` records to continue to reference the Job that they originated from.
 
     An administrator or sufficiently privileged user can manually delete uninstalled Job records if desired, though this will result in the foreign-key from the corresponding `JobResult` and `ScheduledJob` records (if any exist) becoming null. In any case, for tracking and auditing purposes, deleting a Job does **not** automatically delete its related `JobResult` and `ScheduledJob` records.
+
+For any Job that is loaded into Nautobot, the Job must be enabled to run. See [Nautobot Enable Jobs](../../additional-features/jobs.md#enabling-jobs-for-running) for more details.
```
Comment (glennmatthews) [inline nautobot/docs/models/extras/job.md:28]:
```suggestion
For any Job that is loaded into Nautobot, the Job must be enabled to run. See [Enabling Jobs for Running](../../additional-features/jobs.md#enabling-jobs-for-running) for more details.
```

File: nautobot/docs/models/extras/jobbutton.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ The buttons appear at the top right corner of an object's individual detail page
 
 ![Job Button Form](../../media/models/jobbutton_form.png "Job Button Form")
 
+For any Job that is loaded into Nautobot, the Job must be enabled to run. See [Nautobot Enable Jobs](../../additional-features/jobs.md#enabling-jobs-for-running) for more details.
```
Comment (glennmatthews) [inline nautobot/docs/models/extras/jobbutton.md:27]:
```suggestion
For any Job that is loaded into Nautobot, the Job must be enabled to run. See [Enabling Jobs for Running](../../additional-features/jobs.md#enabling-jobs-for-running) for more details.
```

File: nautobot/docs/additional-features/jobs.md (hunk 1 above)
Comment (glennmatthews) [reply nautobot/docs/additional-features/jobs.md:520]: Also, should this be a level-4 heading (`####`) under the `Enabling Jobs for Running` parent heading?

File: nautobot/docs/models/extras/jobhook.md [hunk 4]
```diff
@@ -12,6 +12,8 @@ A Job Hook is a mechanism for automatically starting a [job](../../additional-fe
 * **Enabled** - If unchecked, the job hook will be inactive.
 * **Events** - A job hook may trigger on any combination of create, update, and delete events. At least one event type must be selected.
 
+For any Job that is loaded into Nautobot, the Job must be enabled to run. See [Nautobot Enable Jobs](../../additional-features/jobs.md#enabling-jobs-for-running) for more details.
```
Comment (glennmatthews) [inline nautobot/docs/models/extras/jobhook.md:15]:
```suggestion
For any Job that is loaded into Nautobot, the Job must be enabled to run. See [Enabling Jobs for Running](../../additional-features/jobs.md#enabling-jobs-for-running) for more details.
```

Comment (glennmatthews) [inline nautobot/docs/models/extras/jobhook.md:15]:
```suggestion
For any Job that is loaded into Nautobot, the Job must be enabled to run. See [Enabling Jobs for Running](../../additional-features/jobs.md#enabling-jobs-for-running) for more details.
```

File: nautobot/docs/additional-features/jobs.md (hunk 1 above)
Comment (jvanderaa) [reply nautobot/docs/additional-features/jobs.md:520]: I probably should move it out. That was a copy and paste when I should have been sleeping.

**Review by glennmatthews — APPROVED** [review]

**Review by bryanculver — APPROVED** [review]

**Pull request merged**